pjsip/src/pjsua2/account.cpp

```diff
@@ -1278,5 +1283,7 @@ void Account::sendOutOfDialogRequest(const AccountSendOutOfDialogRequestParam &p
     if (status != PJ_SUCCESS) {
         delete context;
         PJSUA2_RAISE_ERROR(status);
+    } else {
+        context->isAsyncStarted = true;
```

Comment:
How's that work? As I understand crash happened because context already freed, but where does it happened?
Can you please provide some description? It's hard to reason about changes without context.

Reply:
I've just added description about how it works. Please, check

Reply:
Maybe we need to contribute it into original repo?